modules/ref_metering_labels.adoc

```diff
@@ -0,0 +1,13 @@
+[id='runtimes_metering_labels-{context}']
+= {ProductName} Metering Labels for {openshiftshort}
+
+{ProductName} clusters on {openshiftshort} can use the following metering labels:
+
+* `com.redhat.component-name: {component-name}`
+* `com.redhat.component-type: {component-type}`
+* `com.redhat.component-version: {component-version}`
+* `com.redhat.product-name: {product-name}`
+* `com.redhat.product-version: {product-version}`
+
+You can add these labels to {ProductName} pods and then use the {openshiftshort} Metering Operator to determine if your Red Hat subscription is in compliance.
```

Comment:
Should we also link to the metering operator? Not required, just a suggestion.

Reply:
Should we add information on how to add a label to a pod?

Reply:
The linked OpenShift doc has instructions for installing the metering operator. I think that should be fine. Just so I understand, do you mean linking to docs for the operator or actually linking to the operator on the hub somehow?

Reply:
I meant linking to the operator on the hub.  But you're right, the link is in the install docs. That's fine.